conda/core/index.py

```diff
@@ -331,7 +332,53 @@ def maybe_decompress(filename, resp_content):
                              getattr(e.response, 'elapsed', None))
 
 
-def read_local_repodata(cache_path):
+def write_pickled_repodata(cache_path, repodata):
```

Comment:
I know we don't want bare `try/except/pass` constructs. But for now I wanted to make sure that even if the pickle read fails we still get results. We can fix this.